src/components/editor/SinopiaResourceTemplates.jsx

```diff
@@ -30,7 +30,7 @@ class SinopiaResourceTemplates extends Component {
 
       await this.fulfillGroupPromise(groupPromise).then(async () => {
         this.state.groupData.map(group => {
-          const groupName = this.resourceToName(group)
+          const groupName = /[^/]*$/.exec(group)[0]
```

Comment:
I'm OK with moving from a string parsing strategy to a regex matching one, but I wonder whether we should combine strategies here and put the regex strategy **inside** the implementation of `resourceToName` so we don't have an identical regex duplicated several times throughout the code. That will make it less tedious to change later and less error-prone.

Reply:
I ended up using substring inside an imported utilities function.

Reply:
WFM! Thanks.